youtube_dl/extractor/youtube.py

```diff
@@ -1072,6 +1072,9 @@ def add_dash_mpd(video_info):
         else:
             video_categories = None
 
+        m = re.findall(r'''<meta(?:\s+[a-zA-Z0-9:._-]+(?:=[a-zA-Z0-9:._-]+|="[^"]+"|='[^']+'))*?\s+property=['"]?og:video:tag['"]?(?:\s+[a-zA-Z0-9:._-]+(?:=[a-zA-Z0-9:._-]+|="[^"]+"|='[^']+'))*?\s+content=['"]?([^>'"]+?)['"]?\s*>'''
```

Comment:
>$ curl https://www.youtube.com/watch?v=OPf0YbXqDm0 | grep og:video:tag

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   534    0   534    0     0   3581      0 --:--:-- --:--:-- --:--:--  3560        <meta property="og:video:tag" content="Uptown Funk">
        <meta property="og:video:tag" content="Mark">
        <meta property="og:video:tag" content="Ronson">
        <meta property="og:video:tag" content="feat.">
        <meta property="og:video:tag" content="Bruno">
        <meta property="og:video:tag" content="Mars">
        <meta property="og:video:tag" content="Pop">
        <meta property="og:video:tag" content="Columbia">
100  146k    0  146k    0     0   412k      0 --:--:-- --:--:-- --:--:--  412k


What kind of test do you want?